cmd/opampsupervisor/supervisor/supervisor.go

```diff
@@ -1843,6 +1859,7 @@ func (s *Supervisor) onMessage(ctx context.Context, msg *types.MessageData) {
 
 	// Update the agent config if any messages have touched the config
 	if configChanged {
+		span.AddEvent("ConfigChanged")
```

Comment:
```suggestion
		span.AddEvent("Config changed")
```
Just to match the naming of other events.